device-types/Avocent/ACS8008.yaml

```diff
@@ -0,0 +1,41 @@
+---
+manufacturer: Avocent
+model: ACS 8008
+slug: acs8008
+part_number: ACS-8008
+u_height: 1
+is_full_depth: false
```

Comment:
Please add a comment in markdown format with a link to the datasheet if you can. I tried to find one but I am seeing all different model numbers that don't exactly match (i.e. they have more to their names).

Reply:
There are several models of type 8008, so I refined the definition of one of the model + added link to the description